src/Host.cpp

```diff
@@ -766,6 +1267,9 @@ bool Host::installPackage(const QString& fileName, int module)
         pUnzipDialog = dynamic_cast<QDialog*>(loader.load(&uiFile, nullptr));
         uiFile.close();
         if (!pUnzipDialog) {
+            if (pErrorMessage) {
+                *pErrorMessage = QLatin1String("internal error (null pUnzipDialog), please report this to Mudlet developers");
```

Comment:
I'm not a fan of this because years of experience has shown that people will not do that, they at most get frustrated at seeing an error message. Just "not working" causes less frustration. If they want to report this to developers, they will report this either way. So I'd prefer not to have this message in here.

Reply:
I need an error message to pass back to the __nil+message__ bit of the caller if it is a lua function.  It should be a _no way in hell can this happen situation_  and it represents a major internal code breakable if it is reached - so the unhelpfulness of the message is not going to be the thing a user encountering it is _really_ going to be worried about, IMVHO!